tutorials/graphics/crown.C

```diff
@@ -8,23 +8,22 @@
 ///
 /// \author Olivier Couet
 
-TCanvas *crown(){
-   TCanvas *c1 = new TCanvas("c1","c1",400,400);
-   TCrown *cr1 = new TCrown(.5,.5,.3,.4);
+void crown(){
```

Comment:
One possible candidate explanation for the failures is the change in return type of the macro function

Reply:
@couet 
Yes, one need change `tutorials/CMakeLists.txt` file around line 491.
```
set(returncode_1 fit/fit2a.C
                 graphics/arrow.C
                 graphics/crown.C graphics/diamond.C
                 graphics/earth.C graphics/ellipse.C
                 graphics/pavetext.C
```
Macros should be removed from this list.

Reply:
Ah yes ... We do not really need this PR anyway. That was a "clean-up" I made when implementing the IsInside method for TEllipse.... I guess we can close it.